lms/djangoapps/support/static/support/jsx/logged_in_user.jsx

```diff
@@ -176,7 +176,7 @@ LoggedInUser.propTypes = {
     reDirectUser: PropTypes.func.isRequired,
     userInformation: PropTypes.shape({
         course_id: PropTypes.string,
-        username: PropTypes.string,
+        full_name: PropTypes.string,
```

Comment:
```suggestion
        fullName: PropTypes.string,
```

Reply:
You are right we should use camelcase as it is JS code but my intention was different i thought we have python code in html file that is the reason i made it snake case, anyway i have changed it